charts/rancher-vsphere-csi/104.0.1+up3.3.0-rancher2/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: vSphere CSI
+  catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.31.0-0'
```

Comment:
not a blocker: because the upstream chart drops the support for old k8s version < 1.27,  it will be better if the range can match that.

Reply:
I thought about it twice and think we should fix the range to avoid the chart showing up on clusters with unsupported versions. I think UI has some logic depending on this annotation.

Reply:
@thatmidwesterncoder,  I know it is a bit annoying but the proper way to fix the annotation is to fix it on the upstream chart (https://github.com/rancher/vsphere-charts). I can help review PRs.